proto/signer.proto

```diff
@@ -22,9 +22,6 @@ service KeyManagement {
 service Signer {
   // Sign calculates a cryptographic signature using the Key associated with a KeyID and returns the signature
   rpc Sign(SignatureRequest) returns (Signature) {}
-
-  // CheckHealth returns the HealthStatus with the service
-  rpc CheckHealth(Void) returns (HealthStatus) {}
```

Comment:
Why are we removing the `CheckHealth` from this interface?  I realize that we have another health check with the `KeyManagement` service but wasn't sure if we still wanted both